OCPBUGS#25450: Updated images to external load balancer install module

dfitzmau opened the pull request:
[OCPBUGS-25450](https://issues.redhat.com/browse/OCPBUGS-25450)

Version(s):
4.15 to 4.11

Link to docs preview:
* [Installing a cluster on vSphere](https://69453--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned#nw-osp-configuring-external-load-balancer_installing-vsphere-installer-provisioned)
* [Installing a cluster on vSphere with customizations](https://69453--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-customizations#nw-osp-configuring-external-load-balancer_installing-vsphere-installer-provisioned-customizations)
* [Installing a cluster on vSphere with network customizations](https://69453--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-vsphere-installer-provisioned-network-customizations#nw-osp-configuring-external-load-balancer_installing-vsphere-installer-provisioned-network-customizations)
* [Installing a cluster on vSphere in a restricted network](https://69453--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/ipi/installing-restricted-networks-installer-provisioned-vsphere#nw-osp-configuring-external-load-balancer_installing-restricted-networks-installer-provisioned-vsphere)
* [Installer-provisioned postinstallation configuration](https://69453--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_bare_metal_ipi/ipi-install-post-installation-configuration#nw-osp-configuring-external-load-balancer_ipi-install-post-installation-configuration)
* [Networking for OpenStack](https://69453--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/load-balancing-openstack#nw-osp-configuring-external-load-balancer_load-balancing-openstack)

SME review:
- [x] QE has approved this change.

QE review:
- [x] QE has approved this change.
<!--- QE approval is required to merge a PR except for changes that do not impact the meaning of the docs. --->

Additional information:
* https://issues.redhat.com/browse/DXD-496

gpei commented:
@WenXinWei could you please take a look at this?

WenXinWei commented:
> @WenXinWei could you please take a look at this?



> @WenXinWei could you please take a look at this?

sure

WenXinWei commented:
> @WenXinWei could you please take a look at this?

sure

WenXinWei reviewed (commented):
the image quoted here seems wrong.

WenXinWei commented:
the image quoted in Figure 2 and Figure 3 are same. is that expected?

dfitzmau commented:
> the image quoted in Figure 2 and Figure 3 are same. is that expected?

Hi @WenXinWei . My apologies. The error is a mixup on my part. I have now placed the correct image in figure 2's caption.

WenXinWei commented:
hi @dfitzmau.
I think the image and note are not match.
I think the note-->"Figure 1. Example workflow that shows a default external load balancer operating in an OpenShift Container Platform environment" should change to "Figure 1. Example workflow that shows an Ingress Controller operating in an OpenShift Container Platform environment"

"Figure 2. Example workflow that shows an Ingress Controller operating in an OpenShift Container Platform environment"
should change to "Figure 2. Example workflow that shows an OpenShift API operating in an OpenShift Container Platform environment"

"Figure 3. Example workflow that shows an OpenShift API operating in an OpenShift Container Platform environment"
should change to "Figure 3. Example workflow that shows an OpenShift MachineConfig API operating in an OpenShift Container Platform environment"

you can check the image: 
 images/external-load-balancer-default.png, the pod in image is ingress controller. should match "Example workflow that shows an Ingress Controller operating in an OpenShift Container Platform environment"
 images/external-load-balancer-ingress.png, the pod in image is Openshift Api,  should match "Example workflow that shows an OpenShift API operating in an OpenShift Container Platform environment"
 images/external-load-balancer-machine-config-api.png, the pod in image is MachineConfig api , should match "Example workflow that shows an OpenShift MachineConfig API operating in an OpenShift Container Platform environment"

dfitzmau commented:
> hi @dfitzmau. I think the image and note are not match. I think the note-->"Figure 1. Example workflow that shows a default external load balancer operating in an OpenShift Container Platform environment" should change to "Figure 1. Example workflow that shows an Ingress Controller operating in an OpenShift Container Platform environment"
> 
> "Figure 2. Example workflow that shows an Ingress Controller operating in an OpenShift Container Platform environment" should change to "Figure 2. Example workflow that shows an OpenShift API operating in an OpenShift Container Platform environment"
> 
> "Figure 3. Example workflow that shows an OpenShift API operating in an OpenShift Container Platform environment" should change to "Figure 3. Example workflow that shows an OpenShift MachineConfig API operating in an OpenShift Container Platform environment"
> 
> you can check the image: images/external-load-balancer-default.png, the pod in image is ingress controller. should match "Example workflow that shows an Ingress Controller operating in an OpenShift Container Platform environment" images/external-load-balancer-ingress.png, the pod in image is Openshift Api, should match "Example workflow that shows an OpenShift API operating in an OpenShift Container Platform environment" images/external-load-balancer-machine-config-api.png, the pod in image is MachineConfig api , should match "Example workflow that shows an OpenShift MachineConfig API operating in an OpenShift Container Platform environment"

Hi @WenXinWei . Good observation. I updated each diagram accordingly. I also updated some surrounding text (adding the MachineConfig API to the preceeding list).

WenXinWei commented:
/lgtm

dfitzmau commented:
/label peer-review-needed

dfitzmau commented:
Peer reviewer: Clark Hale (SME) is due back on Jan 2nd. Clark has already added a LGTM On https://issues.redhat.com/browse/DXD-496, but I will request a LGTM on #69453 in January.

agantony commented:
/label peer-review-in-progress

agantony commented on modules/nw-osp-configuring-external-load-balancer.adoc:
```diff
@@ -41,10 +39,20 @@ The information and examples in this section are for guideline purposes only. Co
+* OpenShift API
+* OpenShift MachineConfig API
+
+You can choose to configure one or all of these services for an external load balancer. Configuring only the Ingress Controller service is a common configuration option. To better understand each service, view the following diagrams:
```
```suggestion
You can choose whether you want to configure one or all of these services for an external load balancer. Configuring only the Ingress Controller service is a common configuration option. To better understand the individual services, view the following diagrams:
```

agantony reviewed (commented):
Added a suggestion for your consideration; overall lgtm!

agantony commented:
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

agantony commented:
/s OCPBUGS#25450: > OCPBUGS-25450:

agantony commented:
> @dfitzmau: This pull request references [Jira Issue OCPBUGS-25450](https://issues.redhat.com//browse/OCPBUGS-25450), which is invalid:
> 
> * expected the bug to target the "4.16.0" version, but no target version was set
> 
> Comment `/jira refresh` to re-evaluate validity if changes to the Jira bug are made, or edit the title of this pull request to link to a different bug.
> 
> The bug has been updated to refer to the pull request using the external bug tracker.

/s "OCPBUGS#25450:" > "OCPBUGS-25450:"

dfitzmau commented:
> > @dfitzmau: This pull request references [Jira Issue OCPBUGS-25450](https://issues.redhat.com//browse/OCPBUGS-25450), which is invalid:
> > 
> > * expected the bug to target the "4.16.0" version, but no target version was set
> > 
> > Comment `/jira refresh` to re-evaluate validity if changes to the Jira bug are made, or edit the title of this pull request to link to a different bug.
> > The bug has been updated to refer to the pull request using the external bug tracker.
> 
> /s "OCPBUGS#25450:" > "OCPBUGS-25450:" The PR title needs to be updated.

Thanks for the comment. The hash symbol helps alleviate a bug on GH where the hyphen sparks a Jira error message. 

![Screenshot from 2023-12-22 09-45-59](https://github.com/openshift/openshift-docs/assets/57954076/84bd1e54-ded7-44bb-a540-c70cf9a397a6)

clarkhale commented:
Would it be possible to use the work "traffic flow" or "network flow" rather than workflow, in the figure descriptions? @dfitzmau  ?

clarkhale commented:
Looks good to me @dfitzmau.  Thank you!

dfitzmau commented:
/label merge-review-needed

sheriff-rh reviewed (approved):
This looks great, Darragh. Merging!

sheriff-rh commented:
/cherrypick enterprise-4.15

sheriff-rh commented:
/cherrypick enterprise-4.14

sheriff-rh commented:
/cherrypick enterprise-4.13

sheriff-rh commented:
/cherrypick enterprise-4.12

sheriff-rh commented:
/cherrypick enterprise-4.11

sheriff-rh commented:
Looks like you'll need manual cherrypicks for 4.11-4.13 @dfitzmau
